### PR TITLE
Detect color-scheme onload

### PR DESCRIPTION
**Pull request by mgmeyers**
Hi @kepano, thanks for this plugin!

I noticed that the theme doesn't update if the OS color scheme changes while obsidian is closed. This PR fixes that by checking the scheme when obsidian loads.

### CODE REVIEW

File: main.ts [hunk 1]
```diff
@@ -2,7 +2,7 @@ import { App, Workspace, Modal, Notice, Plugin, PluginSettingTab, Setting } from
 
 export default class SystemDarkMode extends Plugin {
 
-	async onload() {
+  async onload() {
```
Comment (mgmeyers) [inline main.ts:5]: Tab to spaces

File: main.ts [hunk 2]
```diff
@@ -48,13 +49,13 @@ export default class SystemDarkMode extends Plugin {
   }
 
   updateDarkStyle() {
-  	document.body.removeClass('theme-light');
+    document.body.removeClass('theme-light');
```
Comment (mgmeyers) [inline main.ts:52]: Tab to spaces

File: main.ts [hunk 3]
```diff
@@ -48,13 +49,13 @@ export default class SystemDarkMode extends Plugin {
...
   updateDarkStyle() {
-  	document.body.removeClass('theme-light');
+    document.body.removeClass('theme-light');
     document.body.addClass('theme-dark');
     this.app.workspace.trigger('css-change');
   }
 
   updateLightStyle() {
-  	document.body.removeClass('theme-dark');
+    document.body.removeClass('theme-dark');
```
Comment (mgmeyers) [inline main.ts:58]: Tab to spaces

File: main.ts [hunk 4]
```diff
@@ -23,7 +23,8 @@ export default class SystemDarkMode extends Plugin {
     // Remove listener when we unload
 
     this.register(() => media.removeEventListener('change', callback));
-
+    
+    callback();
```
Comment (mgmeyers) [inline main.ts:27]: Calling `callback` here so it can detect the OS color scheme on load

**Comment by peyton** [comment]
+1

**Pull request merged**